scripts/update/Updater.php

```diff
@@ -57,9 +57,9 @@ public function update($initialVersion)
             $this->setVersion('2.6.1');
         }
 
-        $this->skip('2.6.1', '4.3.0');
```

Comment:
this update script is not a part of tag? It does not change anything in this PR you can leave it here